domain/consensus/processes/coinbasemanager/coinbasemanager.go

```diff
@@ -3,30 +3,135 @@ package coinbasemanager
 import (
 	"github.com/kaspanet/kaspad/domain/consensus/model"
 	"github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+	"github.com/kaspanet/kaspad/domain/consensus/utils/constants"
+	"github.com/kaspanet/kaspad/domain/consensus/utils/hashes"
+	"github.com/kaspanet/kaspad/domain/consensus/utils/subnetworks"
 )
 
+const scriptPublicKeyMaxLength = 150
```

Comment:
Please move this to the consts file and rename it to coinbaseScriptPublicKeyMaxLength.